list/PageableList.js

```diff
@@ -315,8 +221,87 @@ define([
 			};
 		}),
 
+		/*jshint maxcomplexity:14*/
+		computeProperties: function (props) {
+			if (this.pageLength > 0) {
+				if ("_busy" in props || "hideOnPageLoad" in props || "autoPaging" in props || "showNoItems" in props) {
+					this._updateListView();
+				}
+
+				if ("loadPreviousMessage" in props || "loadingMessage" in props || "_showPreviousPageLoader" in props) {
+					if (this.previousPageLoader && this.previousPageLoader.deliver) {
+						this.previousPageLoader.labels = {
+							loadMessage: string.substitute(this.loadPreviousMessage, this),
+							loadingMessage: this.loadingMessage
+						};
+						this.previousPageLoader.deliver();
```

Comment:
PPS: On further review, I think something like this makes the most sense for the template:

```
<d-page-loader loading={{loadingNextPage}} 
label={{this.loadingNextPage ? this.loadingMessage : this. nextPageButtonLabel}}
on-click={{_load}}>
</d-page-loader>
```

The JS code is responsible for updating a `loadingNextPage` boolean property (to reflect whether or not the load is in progress) as well as the `nextPageButtonLabel` string (based on `loadNextMessage` but with substitution variables replaced).